python/taulabs/uavo.py

```diff
@@ -80,6 +85,29 @@ def from_bytes(cls, data, timestamp, instance_id, offset=0):
 
         return cls._make(field_values)
 
+    def __repr__(self):
+        """ String representation of the contents """
+        rep = self.__class__.__name__ + '('
+        for field in self._fields:
+            field_value = getattr(self, field)
+            # Show ENUMs in a fancy way (string and value)
+            if hasattr(self, 'ENUM_' + field):
```

Comment:
It's probably not needed.  I just don't really like introspection when it's not completely necessary.  Anyways, not a dealbreaker at all.